elm-street.cabal

```diff
@@ -65,20 +65,21 @@ library types
   exposed-modules:     Types
 
   build-depends:       base >= 4.11 && < 4.13
+                     , aeson
+                     , aeson-options ^>= 0.1
```

Comment:
I've created issue #53. Once we implement it, we can drop support for `aeson-options` 👍